builtin/providers/aws/resource_aws_spot_fleet_request.go

```diff
@@ -275,13 +275,20 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"spot_request_state": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"client_token": &schema.Schema{
```

Comment:
Without this, when I try to access either of the fields I get "". The Spot instance resource included this in the schema so that's where I stole it from (https://github.com/hashicorp/terraform/blob/d1ac7d3b2e1be2f32092275340f33a2d73f7b31a/builtin/providers/aws/resource_aws_spot_instance_request.go).